### PR TITLE
update python-sdk to 1.40.0 to fix query-source (now on by default)

### PR DESCRIPTION
**Pull request by ndmanvar**
> we fixed the bug with query source not being attached to some spans & we turned collecting query source on by default in [Python SDK 1.40.0](https://github.com/getsentry/sentry-python/releases/tag/1.40.0).

### CODE REVIEW

File: flask/src/main.py [hunk 1]
```diff
@@ -75,7 +75,6 @@ def __init__(self, import_name, *args, **kwargs):
             traces_sample_rate=1.0,
             before_send=before_send,
             traces_sampler=traces_sampler,
-            enable_db_query_source=True,
```
Comment (realkosty) [inline flask/src/main.py:78]: is this correct?

Comment (ndmanvar) [reply flask/src/main.py:78]: yep. it's defaulted on (:
image
https://github.com/getsentry/sentry-python/releases/tag/1.40.0

**Comment by serglom21** [comment]
Screenshot 2024-01-30 at 6 03 03 PM
Screenshot 2024-01-30 at 6 03 11 PM

Looks good to me! Thanks

**Review by serglom21 — APPROVED** [review]

File: flask/src/main.py (hunk 1 above)
Comment (realkosty) [reply flask/src/main.py:78]: ah gotcha, docs didn't get updated it looks

**Pull request merged**